Xamarin.Forms.Platform.Android/Renderers/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -37,14 +37,22 @@ public ShellFlyoutTemplatedContentRenderer(IShellContext shellContext)
 
         protected virtual void LoadView(IShellContext shellContext)
         {
+
             var context = shellContext.AndroidContext;
             var coordinator = LayoutInflater.FromContext(context).Inflate(Resource.Layout.FlyoutContent, null);
-            var recycler = coordinator.FindViewById<RecyclerView>(Resource.Id.flyoutcontent_recycler);
-            var appBar = coordinator.FindViewById<AppBarLayout>(Resource.Id.flyoutcontent_appbar);
+			var recycler = coordinator.FindViewById<RecyclerView>(Resource.Id.flyoutcontent_recycler);
+			var appBar = coordinator.FindViewById<ViewGroup>(Resource.Id.flyoutcontent_appbar);
+
+			_rootView = coordinator;
+
+			if (recycler == null)
+				recycler = coordinator.FindViewById<RecyclerView>(context.Resources.GetIdentifier("flyoutcontent_recycler", "id", context.PackageName));
+
+			if (appBar == null)
+				appBar = coordinator.FindViewById<ViewGroup>(context.Resources.GetIdentifier("flyoutcontent_appbar", "id", context.PackageName));
```

Comment:
@garuma 

Do you know why when the previewer processes this line the View that it finds is a FrameLayout? Instead of an AppBarLayout? 

It's an AppBarLayout in the layout file

https://github.com/xamarin/Xamarin.Forms/pull/5955/files#diff-36f7448324a68b0a11c71f6372d9c25aR11

Is it just that that's the Android designer is going to replace it with? Because it doesn't want to render an AppBarLayout?